### PR TITLE
maint: remove stray spaces

### PR DESCRIPTION
**Pull request by mathause**
- [x] Passes `isort . && black . && mypy . && flake8`
 - [x] User visible changes (including notable bug fixes) are documented in `whats-new.rst`

Removes stray spaces. Mostly from black removing newlines, e.g.:

```python
raise ValueError("this text was " "formerly on two lines.")
```
also found one potential bug (missing comma)

### CODE REVIEW

File: xarray/core/common.py [hunk 1]
```diff
@@ -577,13 +577,10 @@ def pipe(
 
         >>> def adder(data, arg):
         ...     return data + arg
```
Comment (mathause) [inline xarray/core/common.py:579]:
```suggestion
        ...     return data + arg
        ...
```

File: xarray/core/common.py [hunk 2]
```diff
@@ -577,13 +577,10 @@ def pipe(
...
-        ...
         >>> def div(data, arg):
         ...     return data / arg
```
Comment (mathause) [inline xarray/core/common.py:581]:
```suggestion
        ...     return data / arg
        ...
```

File: xarray/core/common.py [hunk 3]
```diff
@@ -577,13 +577,10 @@ def pipe(
...
-        ...
         >>> def sub_mult(data, sub_arg, mult_arg):
         ...     return (data * mult_arg) - sub_arg
```
Comment (mathause) [inline xarray/core/common.py:583]:
```suggestion
        ...     return (data * mult_arg) - sub_arg
        ...
```

File: xarray/backends/netcdf3.py [hunk 4]
```diff
@@ -20,7 +20,8 @@
     "uint",
     "int64",
     "uint64",
-    "float" "real",
+    "float",
+    "real",
```
Comment (keewis) [inline xarray/backends/netcdf3.py:24]: good catch, that's a bug

**Review by keewis — APPROVED** [review]
thanks, @mathause, looks good to me

**Review by dcherian — APPROVED** [review]
Thanks!

**Pull request merged**